src/azure-cli/azure/cli/command_modules/backup/commands.py

```diff
@@ -44,6 +44,8 @@ def load_command_table(self, _):
 
     with self.command_group('backup container', custom_command_type=backup_custom_base, client_factory=protection_containers_cf) as g:
         g.custom_command('unregister', 'unregister_container', confirmation=True)
+        g.custom_command('re-register', 're_register_wl_container', confirmation=True)
```

Comment:
Can I  think re-register is update?

Reply:
simply running 'register' again is an update. 'Re-register' is different. It is used when we have a 'clone' of the the backed up container i.e., the identity of the machine is the same but physically it is different. So, I have to 're-register' the same container to the vault.

Reply:
Thanks for explanation.